app/src/main/java/com/nextcloud/talk/data/user/model/UserEntity.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ * Nextcloud Talk application
+ *
+ * @author Andy Scherzinger
+ * @author Mario Danic
+ * Copyright (C) 2022 Andy Scherzinger <infoi@andy-scherzinger.de>
+ * Copyright (C) 2017-2020 Mario Danic <mario@lovelyhq.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package com.nextcloud.talk.data.user.model
+
+import android.os.Parcelable
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import com.nextcloud.talk.models.ExternalSignalingServer
+import com.nextcloud.talk.models.json.capabilities.Capabilities
+import com.nextcloud.talk.models.json.push.PushConfigurationState
+import kotlinx.android.parcel.Parcelize
+import java.lang.Boolean.FALSE
+
+@Parcelize
+@Entity(tableName = "User")
+data class UserEntity(
+    @PrimaryKey(autoGenerate = true) @ColumnInfo(name = "id") var id: Long = 0,
+    @ColumnInfo(name = "userId") var userId: String? = null,
```

Comment:
is there a reason why @ColumnInfo is used? It could be omitted when the var name is the same...

Reply:
ok, as i read it's recommended this way to have more flexibility to rename the members without having to rename the DB column..

Reply:
Yes, that is why, because some of the variables might not be as nicely named, so I might change them in the future, but wouldn't migrate the database column names just for that :)